examples/android-material-ui/package.json

```diff
@@ -11,6 +11,7 @@
   ],
   "scripts": {
     "start": "yarn build:stories && storybook dev -p 53743",
+    "start:local": "STORYBOOK_NATIVE_LOCAL_EMULATOR=true yarn build:stories && STORYBOOK_NATIVE_LOCAL_EMULATOR=true storybook dev -p 53743",
```

Comment:
STORYBOOK_NATIVE_LOCAL_EMULATOR=true only needs to be specified once -
this command should be 
```
export STORYBOOK_NATIVE_LOCAL_EMULATOR=true && yarn build:stories && storybook dev -p 53743
```